src/Speech/V1beta1/resources/speech_client_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "interfaces": {
+    "google.cloud.speech.v1beta1.Speech": {
+      "retry_codes": {
+        "retry_codes_def": {
+          "idempotent": [
+            "DEADLINE_EXCEEDED",
+            "UNAVAILABLE"
+          ],
+          "non_idempotent": []
+        }
+      },
+      "retry_params": {
+        "default": {
+          "initial_retry_delay_millis": 100,
+          "retry_delay_multiplier": 1.3,
+          "max_retry_delay_millis": 60000,
+          "initial_rpc_timeout_millis": 60000,
+          "rpc_timeout_multiplier": 1.0,
+          "max_rpc_timeout_millis": 60000,
+          "total_timeout_millis": 600000
+        }
+      },
+      "methods": {
+        "SyncRecognize": {
+          "timeout_millis": 60000,
+          "retry_codes_name": "idempotent",
+          "retry_params_name": "default"
+        },
+        "AsyncRecognize": {
+          "timeout_millis": 60000,
+          "retry_codes_name": "idempotent",
+          "retry_params_name": "default"
+        },
+        "StreamingRecognize": {
```

Comment:
There will be, but not until streaming support is complete (see https://github.com/googleapis/gax-php/pull/51). This is also an issue in pubsub, with the StreamingPull method.

I can remove these manually for now.